src/task/blocking.rs

```diff
@@ -10,21 +62,44 @@ use crossbeam_channel::{bounded, Receiver, Sender};
 use lazy_static::lazy_static;
 
 use crate::future::Future;
+use crate::io::ErrorKind;
 use crate::task::{Context, Poll};
 use crate::utils::abort_on_panic;
+use std::sync::{Arc, Mutex};
```

Comment:
`Arc` unnecessary

Reply:
People tend to share this between threads. It is unnecessary because pool manager doesn't use it. If someone starts to share this across threads we will see the use of it.